_database-integrations/mssql/google-cloud/v1/google-cloud-v1.md

```diff
@@ -0,0 +1,174 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/databases/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Google Cloud SQL Server
+keywords: google-cloudsql-microsoft-sql-server, database integration, etl google-cloudsql-microsoft-sql-server, google-cloudsql-microsoft-sql-server etl
+permalink: /integrations/databases/google-cloudsql-microsoft-sql-server
+summary: "Connect and replicate data from your Google Cloud SQL Server database using Stitch's Google Cloud SQL Server integration."
+
+show-in-menus: true
+key: "google-cloudsql-microsoft-sql-server-integration"
+
+# -------------------------- #
+#     Integration Details    #
+# -------------------------- #
+
+name: "google-cloudsql-microsoft-sql-server"
+display_name: "Google Cloud SQL Server"
+
```

Comment:
There should be a `setup-name` attribute here as there isn't an actual CloudSQL MSSQL integration in the app. Users will have to use the `Microsoft SQL Server` integration to do this.

Reply:
To clarify - this should be the **display name** of whatever integration they'll need to use to set up the integration in Stitch. It's at the end of [this section](https://docs-about-stitch-docs.netlify.app/reference/integration-templates/databases/#stitch-details) of our metadocs if you need some more guidance.